js/projects.js

```diff
@@ -0,0 +1,177 @@
+/*!
+ * projects.js by Yi Liu - @yiliu
+ * Copyright 2020 Eclipse Foundation
+ * http://www.eclipse.org/org/documents/epl-v10.php
+ */
+
+(function($, window, document) {
+    $(function() {
+
+        $.ajax({
+            type: "GET",
+            url: "/js/projects.json",
+            dataType: "json",
+            cache: true,
+            success: function(data) {
+
+                var projectInfos = [];
+
+                var val = 20;
+                $('.progress-bar').css('width', val + '%').attr('aria-valuenow', val);
+
+                var i = 0;
+
+                $.each(data, function(key, value) {
+                    
+                    if (value.state === "Archived") {
+                      return true;
+                    }
+                  
+                    i++;
+                    var val = 20 + (i / Object.keys(data).length) * 80;
+
+                    $('.progress-bar').css('width', val + '%').attr('aria-valuenow', val);
+                    var title = stringJanitor(value.name);
+                    var id = stringJanitor(value.project_id);
+                    var link = value.url;
+                    if (!validateUrl(link)) {
+                        link = "http://projects.eclipse.org/projects/" + id;
+                    }
+                    var desc = stringJanitor(value.summary, {
+                        "cut": true,
+                        "ellipsis": ' [&hellip;] <br><a href="' + link + '"> Read more&hellip;</a>'
+                    });
+
+                    if (id == 'rt.ecf' || id == 'tools.mtj' || id == "technology.koneki" || id == "tools.titan" || id == "iot.tiaki"  || id == "technology.sensinact"  || id == "technology.iottestware"  || id == "iot.krikkit" || id == "technology.uomo" || id == "iot.risev2g")
+                        return true;
```

Comment:
m not quite sure about what happened with those projects ids, probably not needed for ecd. Gonna remove later...

Reply:
+1 let's remove anything that is specific to iot.